docs/DeveloperGuide.adoc

```diff
@@ -844,13 +882,34 @@ Use case ends.
 +
 Use case resumes at step 2.
 
-_{More to be added}_
+=== Use case: Search a person by tag
+*MSS*
+
+1. User requests to search and provides search params
+2. Addr	essBook shows a list of person fitting the given criteria
++
+Use case ends.
+
+=== Use case: View Linkedin profile of a person
+*MSS*
+
+1. User requests to search the person by name
+2. AddressBook shows a list of person
+3. User chooses a person by index in the list
+4. AddressBook shows the linkedin page of that person
++
+Use case ends.
```

Comment:
The given use cases are fine, although it would have been nice to choose a slightly more complicated feature so that you can also write extensions in addition to the main success scenario.